.gitignore

```diff
@@ -36,6 +36,7 @@ venv
 
 # Static documentation site generated by Mkdocs
 site
+!docs/overrides/*.html
```

Comment:
Why do we need an override to be included in git. Is it being ignored somewhere else?

Reply:
If it really needs to be in this file, a comment would be useful for future contributors.

Reply:
Good catch, remove it in the latest commit.